lib/matplotlib/animation.py

```diff
@@ -1105,3 +1108,5 @@ def _draw_frame(self, framedata):
         # Call the func with framedata and args. If blitting is desired,
         # func needs to return a sequence of any artists that were modified.
         self._drawn_artists = self._func(framedata, *self._args)
+        for a in self._drawn_artists:
+            a.set_animated(self._blit)
```

Comment:
Isn't this going to wreak havoc on the stale-handling code? Don't we want to avoid setting artists as stale within drawing code paths?